protocol/app/testdata/default_genesis_state.json

```diff
@@ -455,7 +455,16 @@
     "total_escrowed": []
   },
   "upgrade": {},
-  "vault": {},
+  "vault": {
+    "params": {
+      "layers": 2,
+      "spread_min_ppm": 3000,
+      "spread_buffer_ppm": 1500,
+      "skew_factor_ppm": 500000,
+      "order_size_ppm": 100000,
+      "order_expiration_seconds": 5
```

Comment:
i would recommend this making the `order_expiration_seconds` = 2, since this is more in line with a single block time.

Reply:
each block, we cancel pervious block's order and place new ones. but yeah having it as `2` makes sense